### DISCUSSION TITLE
¿En donde se encuentran los videos que el MC sube?

### DISCUSSION THREAD

**Discussion by LeonardoRosalesG** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#258] [Q&A] [answered]
Tengo duda de si ya los subió a algún lado, porque no he visto rastro de ellos.
No me refiero a los del live, sino a los extras.

Gracias por su atención. :)

**Comment by carlos-gmx3** [comment, marked as answer]
Como recordatorio, el MC puso un link a los videos en los diferentes canales por semana que ha hecho en Microsoft Teams
![imagen](https://user-images.githubusercontent.com/99064239/156902789-e54f597f-e116-4586-a11b-2b6587ade4e8.png)

También puedes buscarlos por tu cuenta ingresando a office.com con tu cuenta de launch-X
![imagen](https://user-images.githubusercontent.com/99064239/156902826-157c4f7a-bb08-4ba6-8ea0-5e8369a1e520.png)

Dale click en los puntos y busca "Stream", ya sea solo o en todas las aplicaciones
![imagen](https://user-images.githubusercontent.com/99064239/156902863-79693ad9-c128-46b8-aebd-8f4e6f27c25a.png)

En la pestaña "Encontrar" le das click en la opción que dice "Videos y listo"
![imagen](https://user-images.githubusercontent.com/99064239/156902914-7378b0ed-71da-469b-8829-59c3f03b888e.png)

**Reply by LeonardoRosalesG** [reply]
Gracias, pero no me refería a esos. 
En varios lives ha mencionado que tiene planeado subir contenido extra que no se ve en los lives, a esos videos me refería.

**Reply by JaquelineJT** [reply]
Están en el github en la sección de temario, los fue agregando a las lecciones en la parte de arriba
![image](https://user-images.githubusercontent.com/73417902/156903585-cc0fc920-eed0-4495-b257-3a83ed47c2a4.png)

**Reply by LeonardoRosalesG** [reply]
Muchas gracias. Con tu respuesta y la de  @carlos-gmx3 se aclaró mi duda.

**Reply by carlos-gmx3** [reply]
Tooooooodos los videos, están ahí mismo, todos
![imagen](https://user-images.githubusercontent.com/99064239/156905361-b60daee8-9755-4dd6-b328-4f807e135fa3.png)
